.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   - BUILD_ARCH=arm
 before_install: go get github.com/Masterminds/glide
 install: glide install
-script: GOARCH=$BUILD_ARCH GOOS=$BUILD_OS go build
+script: GOARCH=$BUILD_ARCH GOOS=$BUILD_OS go build -ldflags '-X main.version='$TRAVIS_TAG
```

Comment:
It probably needs a comment - I could not figure out how to get go build to pull in env vars inside the quote marks, putting the TRAVIS_TAG outside the quote marks works fine.